api/serializers.py

```diff
@@ -27,7 +27,7 @@ class Meta:
             "is_staff",
             "is_superuser",
             "perms",
-            "groups",
```

Comment:
this generated an error when the backend had groups defined. Because groups has no serializer and viewset defined